src/sidebar/tabcenter.js

```diff
@@ -14,11 +14,11 @@ TabCenter.prototype = {
 
     this._setupListeners();
 
+    const {id: windowId} = await windowPromise;
```

Comment:
Is there a reason behind this change?

Reply:
set _themeIntegration needs the windowId.